README.md

```diff
@@ -8,7 +8,7 @@
 
 ## What it does ?
 
-It fetches all the **eligible expired pools** created using [Diva Protocol App](https://app.diva.finance/) from the [Diva Graph](https://thegraph.com/hosted-service/subgraph/divaprotocol/diva-goerli-new). 
+It fetches all the **eligible expired pools** created using [Diva Protocol App](https://app.diva.finance/) from the [Diva Graph](https://thegraph.com/hosted-service/subgraph/divaprotocol/diva-goerli-new-2). 
```

Comment:
Please replace the DIVA Graph link with the following https://thegraph.com/hosted-service/subgraph/divaprotocol/diva-polygon-pilot